release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3116,6 +3116,68 @@ $ ccoctl azure create-all \
 --
 (link:https://issues.redhat.com/browse/OCPBUGS-22651[*OCPBUGS-22651*])
 
+* When installing an {product-title} cluster with static IP addressing and Tang encryption, worker nodes start without network settings. This condition prevents worker nodes from accessing the Tang server, causing installation to fail. To address this condition, you must set the network settings for each worker node as `ip` kernel arguments.
```

Comment:
according to what i understood this need to be added for all nodes (also masters)